canonical-websites/build.snapcraft.io
# Migrated headings components to vanilla styles

## Done

Removed old Heading component, replaced with component with vanilla generated css module.

## QA

- Check out this feature branch
- Run the site using the command ```npm start -- --env=environments/dev.env```
- View the site locally in your web browser at: [http://0.0.0.0:8000/](http://0.0.0.0:8000/)
- Headings should have Vanilla styles


## Issue / Card

Fixes #889
